eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/push/AsyncHTTPPushRequest.java

```diff
@@ -328,7 +328,7 @@ ClientRetCode processResponseContent(String content) {
             Map<String, Object> ret =
                 JsonUtils.parseTypeReferenceObject(content, new TypeReference<Map<String, Object>>() {
                 });
-            Integer retCode = (Integer) ret.get("retCode");
+            Integer retCode = (Integer) Objects.requireNonNull((ret.get("retCode")));
```

Comment:
I think it should be done like this:
Integer retCode = (Integer) Objects.requireNonNull(ret).get("retCode");

Reply:
Hi @Alonexc, I tried this first but it was not giving an error. Therefore I thought to go with this way.
I have made the review changes. Can you please review it.

Reply:
There is a suggestion to change lines 221-241 to use switch. e.g:
```
switch (result) {
                        case OK:
                        case REMOTE_OK:
                        case FAIL:
                            complete();
                            if (isComplete()) {
                                handleMsgContext.finish();
                            }
                            break;
                        case RETRY:
                        case NOLISTEN:
                            delayRetry();
                            if (isComplete()) {
                                handleMsgContext.finish();
                            }
                            break;
                        default: // do nothing
                    }
```

Reply:
Hi @Alonexc , made the required changes Please check